### DISCUSSION TITLE
My student status expired and cannot apply reapply

### DISCUSSION THREAD

**Discussion by hkumaster** [community/community#119964] [GitHub Education]
### Hi everyone,

My student package recently expired and I want to apply it again
So I go to the
https://education.github.com/discount_requests/additional_information
page and want to submit my ID card, it just keep routing me back to the same page and nothing update. I am not sure whats happen but seems like I cannot apply. I will be appreciated if anyone can tell me what is happen and what can I do to apply.
![recording](https://github.com/community/community/assets/98153187/b35de5f0-4501-4a81-bd45-ea10e7f950c3)

**Comment by Mi-Zel** [comment]
Same thing happens to me.

**Reply by dabzse** [reply]
did  you check the dimensions and the size?
I can upload my, but this perfect system now can't accept it (but it does in 2022)
